docs/section12/Rule12-1.md

```diff
@@ -1,36 +1,38 @@
-
-# Receptacle - Rule 12-1
-
+# Receptacles – Rule 12-1   
+**Schema Version:** 0.0.37        
+**Mandatory Rule:** True          
 **Rule ID:** 12-1  
-**Rule Description:** Number of spaces modeled in User RMR and Baseline RMR are the same  
-**Rule Assertion:** Baseline RMR = User RMR  
-**Appendix G Section:** Section Table G3.1-12 Modeling Requirements for the Proposed design  
-**Appendix G Section Reference:** None  
-
-**Applicability:** All required data elements exist for B_RMR and U_RMR  
-**Applicability Checks:** None  
-**Manual Check:** None  
-**Evaluation Context:** Each Data Element  
+**Rule Description:** Receptacle and process power shall be modeled as identical to the proposed design  
+**Rule Assertion:** B-RMD = P-RMD                                           
+**Appendix G Section:** G3.1  
+**Appendix G Section Reference:** Table G3.1-12  
 **Data Lookup:** None  
-
-## Rule Logic:  
-
-- Get the total number of spaces in the building segment in the User model: ```for building_segment_user in U_RMR.building.building_segments:```  
-
-  - For each thermal_block in building segment: ```thermal_block_user in building_segment_user.thermal_blocks:```
-
-  - For each zone in thermal block: ```zone_user in thermal_block_user.zones:```
-
-  - For each space in zone, add to the total number of spaces in the User model: ```for space_user in zone_user.spaces: num_of_spaces_user += 1```  
-
-- Get the total number of spaces in the building segment in the Baseline model: ```for building_segment_baseline in B_RMR.building.building_segments:```  
-
-  - For each thermal_block in building segment: ```thermal_block_baseline in building_segment.thermal_blocks:```
-
-  - For each zone in thermal block: ```zone_baseline in thermal_block_baseline.zones:```
-
-  - For each space in zone, add to the total number of spaces in the Baseline model: ```for space_baseline in zone_baseline.spaces: num_of_spaces_baseline += 1```  
-
-**Rule Assertion:** ```num_of_spaces_user == num_of_spaces_baseline```  
-
-**[Back](../_toc.md)**
+**Evaluation Context:** Each RPD  
+
```

Comment:
I'm curious about the choice of evaluation context here.

I've had many projects where credit was allowed for Energy Star equipment in kitchens or laundries (or other allowed misc equipment reductions), but most of the equipment in the building was the same.  The logic here looks great, just want to make sure the end-user gets the information about which equipment is compliant vs not.  We apply rules about surfaces to each surface, so why not here?

Reply:
The choice was made to minimize the number of evaluations and messages that are reported - I am anticipating that we may be at a point where we are providing so many messages that they no longer will be useful because there are so many. If the implementation is done in a way that shows the space ID's which cause an UNDETERMINED outcome would that address your concern?

Reply:
I definitely see the concern, and I think this has a lot to do with how the output report is displayed.  For example, a user should be able to filter by rule, evaluation result (PASS, FAIL, UNDETERMINED), or even by the raise_message string.

What you're suggesting works for this rule, but it means that this rule is different from the other rules, so it'll make creating a concise tool tool to show data more difficult.

The issue you've highlighted is much bigger than will be solved by changing the approach with this one rule.  I think it's better handled with some filtering capabilities in a result viewer tool

Reply:
There are rules that exist that do it both ways. Here is an example of a rule that is evaluated once and lists the ID's that caused the outcome.

![image](https://github.com/pnnl/ruleset-checking-tool/assets/122476654/ef16c6ca-6f61-4c81-b3b9-f0af6045bff9)